tsconfig.json

```diff
@@ -6,11 +6,11 @@
 		"module": "node16",
 		"moduleResolution": "node16",
 		"moduleDetection": "force",
-		"target": "ES2021", // Node.js 16
+		"target": "ES2022", // Node.js 18
 		"lib": [
 			"DOM",
 			"DOM.Iterable",
-			"ES2021"
+			"ES2023"
```

Comment:
Why doesn't `lib` match `target`?

Reply:
On node.green we can check which version we can use

Reply:
So why not also 2023 "target" then?

Reply:
Yeah, was just providing a source to pick one

Reply:
I matched to tsconfig to [@tsconfig/node18](https://www.npmjs.com/package/@tsconfig/node18). Looking into it ES2023 has more than whats listed on node.green:

> ECMAScript 2023, the 14th edition, introduced the toSorted, toReversed, with, findLast, and findLastIndex methods on Array.prototype and TypedArray.prototype, as well as the toSpliced method on Array.prototype; added support for #! comments at the beginning of files to better facilitate executable ECMAScript files; and allowed the use of most Symbols as keys in weak collections.

Source: https://tc39.es/ecma262/#sec-intro

For example `[].toSorted()` works with ES2023 but does not in Node.js 18.

This means only ES2022 is supported by Node.js 18 and node.green seems to miss some features? I also can not find top-level await from ES2022 on there.